src/components/EDraggable.vue

```diff
@@ -50,22 +53,10 @@ const { x, y, style, isDragging } = useDraggable(draggableRef, {
 const snappedX = computed(() => Math.round(x.value / tileSize.value));
 const snappedY = computed(() => Math.round(y.value / tileSize.value));
 
-watch(draggable, (newValue, oldValue) => {
-  isMinimised.value = newValue.isMinimised ? newValue.isMinimised : false;
-  order.value = newValue.order;
-  gridPosX.value = newValue.gridPosX ? newValue.gridPosX : 0;
-  gridPosY.value = newValue.gridPosY ? newValue.gridPosY : 0;
-  // @TODO: Why does dis not work?
```

Comment:
Not sure. Is it about deep object compare? what is newValue / onValue type? Ref?

Reply:
This part is deleted. What lines are you reffering to?